tools/validators/instance_validator/tests/fake_instances/BAD/translation_invalid_value_range_format.yaml

```diff
@@ -12,20 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# good translation with present value adhereing to udmi
-
-SDC_EXT-17-GUID:
-  type: HVAC/SDC_EXT
-  code: SDC_EXT-17
+US-SEA-BLDG1-GUID:
+  type: FACILITIES/BUILDING
+  code: US-SEA-BLDG1
   cloud_device_id: "foobar"
   translation:
-    shade_extent_percentage_command:
-      present_value: "points.brightness_percentage_sensor_66679452.present_value"
```

Comment:
this might be the correct one?

Reply:
Done. Changed units -> points.zone_air_temperature_sesnor.units